packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -547,6 +565,12 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
         uint256 recipientCredit = credit[_recipient];
         credit[_recipient] = 0;
 
+        // Try to withdraw the WETH amount so it can be used here. Don't bother doing this if the
+        // recipient has no credit. We'll still trigger the external call though.
+        if (recipientCredit > 0) {
```

Comment:
Why do we perform the external call if the credit is 0? Any reason we don't just revert if there's nothing to claim?

Reply:
No idea, this was part of the original behavior in this. Changing this broke a bunch of tests so I figured it was better to leave it out of scope. cc @clabby

Reply:
Fixed this up to revert for zero credit claims and updated the tests to expect these cases. 👍